kitsune/sumo/forms.py

```diff
@@ -20,32 +17,22 @@ def __init__(self, *args, **kwargs):
     def clean(self, *args, **kwargs):
         """Generic clean method used by all forms in the question app.
 
-        Parse content for any toll free numbers.
+        Parse content for suspicious content.
+        - Toll free numbers
+        - Toll vanity numbers
```

Comment:
Again, strictly speaking we'll match obfuscated toll free numbers and any well formed number in the NANP